rc/filetype/conf.kak

```diff
@@ -1,4 +1,4 @@
-hook global BufCreate .+\.(repo|cfg|properties|desktop) %{
+hook global BufCreate .+\.(conf|repo|cfg|properties|desktop) %{
```

Comment:
This will conflict with various other highlighters, such as the resolv.conf one in etc.kak or the exherbo ones, we would need some way to give priority to one or the other.

Reply:
@mawww  is there a use case in particular for say the exherbo files needed for config or the resolve.conf in etc.kak? Does ti have anything to do with the mime encoding at the time it gets written? I just don't see it. And if I don't see it, it becomes a problem.  Is there another outstanding issue related to this that I can read about?

Reply:
@mawww  undone. 

And I went ahead and added nginx.kak. Although I agree with you in that if by any chance there's a problem, conflict with whatever other files from the list, nginx however, needs highlighting no less than the rest. Or so I think.